src/provider/controller/NotificationCenter.ts

```diff
@@ -131,6 +133,14 @@ export class NotificationCenter extends AsyncInit {
         });
     }
 
+    private async hideWindowOffscreen() {
+        const {window} = this._webWindow;
+        const {virtualScreen} = await fin.System.getMonitorInfo();
+        const height = virtualScreen.bottom;
+        await window.showAt(virtualScreen.left - NotificationCenter.WIDTH * 2, virtualScreen.top - height * 2);
```

Comment:
The y position is `virtualScreen.top - (virtualScreen.bottom * 2)`?

Should `height` be the window height (as in, `primaryMonitor`, rather than `virualScreen`)?

Reply:
Switched to `primaryMonitor.availableRect.bottom`.

Reply:
Is the primary monitor always based at (0, 0)? :-P

Reply:
If we use `primaryMonitor` isnt there a chance that the flash will occur on a monitor that isnt the primary?  I would imagine using left/top of the virtual screen would clear all screens

Reply:
Not 100%, but I think the virtual screen coordinates are based on the primary monitor bounds?

@ryanpetersonOF we only use the primary monitor to get the height of the center and subtract that from the top of the virtual screen meaning it can't flash on a monitor.